presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -159,9 +160,11 @@ public Statement rewrite(
             Optional<QueryExplainer> queryExplainer,
             Statement node,
             List<Expression> parameters,
-            AccessControl accessControl)
+            AccessControl accessControl,
+            WarningCollector warningCollector)
     {
-        return (Statement) new Visitor(metadata, parser, session, parameters, accessControl, queryExplainer).process(node, null);
+        return (Statement) new Visitor(metadata,
```

Comment:
Remove newline. There are lines in this class that have same/similar length. If you think it's too long, please put each parameter on a different line.